Vegetarian-Food-Finder/Api/ApiManger.swift

```diff
@@ -2,8 +2,35 @@
 //  ApiManger.swift
 //  Vegetarian-Food-Finder
 //
-//  Created by Abdullahi Ahmed on 7/6/22.
+//  Created by Abdullahi Ahmed on 7/7/22.
 //
 
 import UIKit
 
+final class ApiManger {
```

Comment:
This Api is handling Yelp requests only. Let's name it closer to what it does: for example `YelpManager`. Also there's a typo in `Manger`.